examples/simple/bookinfo.yaml

```diff
@@ -21,6 +21,7 @@ spec:
           #   name: <secret-name>
           #   namespace: <secret-namespace>
         release:
+          timeout: 10m
```

Comment:
Just so that we have an explicit timeout specified in the sample applicationgroup